src/code.c

```diff
@@ -201,6 +180,24 @@ SEXP method_call_(SEXP call, SEXP generic, SEXP envir) {
     R7_method_lookup_error(generic, signature_classes, envir);
   }
 
+  // Now add the remaining, non-dispatched, args
```

Comment:
The loop above builds up the argument list for all arguments that are dispatched on. Now we need to do the same thing for the remaining arguments.